### DISCUSSION TITLE
watching repos and defaults for new repos

### DISCUSSION THREAD

**Discussion by iantaylor-NOAA** [NOAA-FIMS/FIMS#637] [General]
A link in a recent issue alerted me to the existence of a new https://github.com/NOAA-FIMS/seaside-chats repo that @kellijohnson-NOAA created a few weeks ago. However, @kellijohnson-NOAA was the only one watching the repo which means any questions posted in an issue would not be seen by anyone other than Kelli. This is unlike the setup of the `pfmc-assessment` github organization that I'm familiar with, where we all 15 members are set to watch repos by default, the FIMS settings don't automatically have us watch anything.

Perhaps when new repos are created, a discussion message could be posted or some other way to make folks aware that they should opt in to watch if they want. Or we could figure out how to make watching the default for everyone and folks would have to opt out of the repos that they don't want notifications for.

**Comment by kellijohnson-NOAA** [comment]
Thanks @iantaylor-NOAA for noting that, I will add a link to the repository in next week's FIMS Weekly and I should have alerted everyone earlier. Unless someone objects, I think that we should keep the non-following status of new repositories as the default and use the Discussion board, as you note here, or the FIMS Weekly to announce repositories that people should "think" about following. Thanks again for noticing!

**Comment by kellijohnson-NOAA** [comment]
I wanted to bring it to everyone's attention again that it is **NOT** the default for individuals in an organization to automatically watch a new repository. So, if you create a NOAA-FIMS repo and want people to know about it, you must add them, tag them, or make a post here letting people know about the new repo. Thanks!
